static/src/javascripts/projects/common/modules/navigation/newHeaderNavigation.js

```diff
@@ -0,0 +1,152 @@
+// @flow
+
+import qwery from 'qwery';
+import fastdom from 'fastdom';
+import ophan from 'ophan/ng';
+import userAccount from 'common/modules/navigation/user-account';
+
+const html = qwery('html')[0];
```

Comment:
Removal of `qwery` will be part of another PR which is already in the pipeline :)